ha/core/event_analyzer/filter/filter.py

```diff
@@ -41,17 +49,25 @@ def filter_event(self, msg: str) -> bool:
         """
         pass
 
+    @staticmethod
+    def get_msg_type(msg: dict) -> str:
+        """
+        Check if the msg type is iem or alert
+        """
+        msg_type = msg.get("sensor_response_type")
+        return msg_type["info"]["resource_type"]
```

Comment:
Move these literals to const. Check other places too.
Create a class for AlertConstants and IemConstants.

Reply:
Done.